hazelcast/password.txt

```diff
@@ -0,0 +1 @@
+h4z3lc4$t
```

Comment:
file added accidentally (and change your password 😉 )

Reply:
oops, i dont actually know the purpose of that password file

Reply:
removed!